RetakesAllocator/Helpers.cs

```diff
@@ -160,10 +161,10 @@ public static bool IsWarmup()
     {
         return GetGameRules().WarmupPeriod;
     }
-    
-    public static bool IsFreezeTime()
+
+    public static bool IsWeaponAllocationAllowed()
     {
-        return GetGameRules().FreezePeriod;
+        return Configs.GetConfigData().AllowAllocationAfterFreezeTime || GetGameRules().FreezePeriod;
```

Comment:
I feel like this logic is slightly wrong

Reply:
What I was thinking is:
- If you allow allocations after freeze time, then you "always" allow them
- Otherwise, you only allow allocations during the freeze period

Is there better logic?

Reply:
I think it should be
```
public static bool IsWeaponAllocationAllowed()
{
    return Configs.GetConfigData().AllowAllocationAfterFreezeTime || !GetGameRules().FreezePeriod;
}
```

Reply:
If `AllowAllocationAfterFreezeTime=false`, and `GetGameRules().FreezePeriod=true` - meaning you dont want to allow weapon command after freeze time, but we are currently in freeze time - then this would evaluate to `false || !true` -> `false || false` -> `false`, which is the opposite of what we want

Reply:
I wrote some tests that demonstrate what I think is the expected behavior
https://github.com/yonilerner/cs2-retakes-allocator/pull/28/files#diff-5759329b7f8958a057dfe2c40ebaec417e95a76ce19fd949628c384c90ba2ff4R15-R19